cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -0,0 +1,59 @@
+function createNewDashboard(dashboardName) {
+  cy.visit('/dashboards');
+  cy.getByTestId('CreateButton').click();
+  cy.get('li[role="menuitem"]')
+    .contains('Dashboard')
+    .click();
+
+  cy.server();
+  cy.route('POST', 'api/dashboards').as('NewDashboard');
+
+  cy.getByTestId('EditDashboardDialog').within(() => {
+    cy.getByTestId('DashboardSaveButton').should('be.disabled');
+    cy.get('input').type(dashboardName);
+    cy.getByTestId('DashboardSaveButton').click();
+  });
+
+  return cy.wait('@NewDashboard').then((xhr) => {
+    const slug = Cypress._.get(xhr, 'response.body.slug');
+    assert.isDefined(slug, 'Dashboard api call returns slug');
+    return Promise.resolve(slug);
```

Comment:
I think you can just return `slug` from here

Reply:
You're right!